src/data/animations/anim-config-schema.ts

```diff
@@ -225,32 +312,36 @@ const animTimedEventSchema: Schema = {
     bgX: {
       type: "number",
       default: 0,
+      nullable: true,
     },
 
     /** The y-coordinate of the background image */
     bgY: {
       type: "number",
       default: 0,
+      nullable: true,
     },
 
-    /** The amount of time the image is displayed (ms) */
+    /** The amount of time the image is displayed (in frames) */
```

Comment:
NIT:
```suggestion
    /** The duration the image is displayed (in frames) */
```
Using `time` and `(in frames)` is kind of contradicting.